lib/graphql/schema_comparator/changes.rb

```diff
@@ -1,6 +1,21 @@
 module GraphQL
   module SchemaComparator
     module Changes
+      module SafeTypeChange
+        def safe_change?(old_type, new_type)
+          if !old_type.kind.wraps? && !new_type.kind.wraps?
+            old_type.name == new_type.name
```

Comment:
Can we simply use `old_type == new_type` here? I know it's the equivalent right now, but if eventually something is added to `TypeName` nodes we would already cover this case with `==`!

Reply:
Yeah, works 👍

Reply:
8D